runtime/jcl/common/system.c

```diff
@@ -357,10 +357,37 @@ jobject getPropertyList(JNIEnv *env)
 		strings[propIndex++] = !usernameAlloc || result ? "unknown" : usernameAlloc;
 	}
 
-#undef USERNAME_LENGTH
+	{
+		/* buffer to hold the size of the maximum direct byte buffer allocations */
+		char maxDirectMemBuff[24];
+#if defined(OPENJ9_BUILD)
```

Comment:
Rather than duplicating the code here, would it be better to ifdef around the initial -1 check, then do the -1 check again outside the ifdef?

```ifdef (open)
if (-1) set to 7/8
endifdef
if (!-1) set property
'''